src/js/components/Heading/index.d.ts

```diff
@@ -1,8 +1,9 @@
 import * as React from "react";
 import { 
   A11yTitleType,
-  AlignSelfType,
+  AlignSelfType, 
```

Comment:
can you check why you are adding empty space after a comma? I've seen it in a couple of places.